CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and **Merged pull requests**. Critical items to know are:
 The versions coincide with releases on pip. Only major versions will be released as tags on Github.
 
 ## [0.0.x](https://github.com/oras-project/oras-py/tree/main) (0.0.x)
+ - bugfix maintain requests's verify valorization for all invocations, augment basic auth header to existing headers (0.2.0)
```

Comment:
Looks like this is also 0.2.0 (your addition is correct!) but let's indent the line below it to be a sub-bullet, and remove the second mention of 0.2.0.

Reply:
Will do! 👍

Reply:
done with f8487521aa4264b2e83a72e7375e3fa1fcc9c2f2 🚀